symplyphysics/laws/optics/interference_from_two_holes.py

```diff
@@ -0,0 +1,42 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+                           validate_input, validate_output)
+
+# Description
+## The coherence of vibrations that occur at the same
+## time at different points in the plane perpendicular to the direction
+## of wave propagation is called spatial coherence.
+## Spatial coherence depends on the conditions of radiation and
+## the formation of light waves in extended sources.
```

Comment:
It looks like a poorly translated description. Could you try to look for some better description?